app/controllers/tutorials_controller.rb

```diff
@@ -2,5 +2,6 @@ class TutorialsController < ApplicationController
   def show
     tutorial = Tutorial.find(params[:id])
     @facade = TutorialFacade.new(tutorial, params[:video_id])
+    @tutorial = Tutorial.last
```

Comment:
As with some of our other controllers, might be nice if we didn't have to pass two instance variables to the view